docs/start/install.mdx

```diff
@@ -63,7 +63,7 @@ Whether you will work locally or in a cloud environment, the first step for all
 1. Install the following packages.
 
       <Admonition type="note">
-          Periodically rerun these commands or check the [Qiskit release notes](https://qiskit.org/documentation/release_notes.html) and the [Qiskit Runtime release notes](https://qiskit.org/ecosystem/ibm-runtime/release_notes.html) so that you always have the latest versions. 
+          Periodically rerun these commands or check the [Qiskit release notes](https://docs.quantum-computing.ibm.com/api/qiskit/release-notes) and the [Qiskit Runtime release notes](https://docs.quantum-computing.ibm.com/api/qiskit-ibm-runtime/release-notes) so that you always have the latest versions. 
```

Comment:
```suggestion
          Periodically rerun these commands or check the [Qiskit release notes](../api/qiskit/release-notes) and the [Qiskit Runtime release notes](../api/qiskit-ibm-runtime/release-notes) so that you always have the latest versions. 
```